Makefile

```diff
@@ -22,6 +22,10 @@ build: coq_boot
 check: coq_boot
 	dune build $(DUNEOPT) @check
 
+.PHONY: test
+test: build
```

Comment:
At some point this could be moved to dune, which is usually better in terms of rebuilds.

We would ideally makes the deps a bit finer grained here, so tests run much faster. For example:
```
test/node_modules: test/package.json
     cd test && npm i

test: test/node_modules coq-lsp.install vendor/coq/coq-stdlib.install
      cd test && npm test
```
etc, you get the idea folks.